packages/nollup/src/index.js

```diff
@@ -0,0 +1,166 @@
+const NAMESPACE = '__PREFRESH__';
+
+module.exports = function() {
+	return {
+		resolveId(id) {
+			if (id === 'prefresh-runtime') {
```

Comment:
Where do we get this `id` from? Since in our case this will be `@prefresh/core` or am I misunderstanding what this is

Reply:
Virtual module that's defined in ```load``` hook that contains the common ```module.hot``` logic. I updated the PR to put this into a separate file to make the flow less confusing and easier to maintain. :)